monkey/monkey_island/cc/server_utils/encryption/encryptors/password_based_string_encryptior.py

```diff
@@ -0,0 +1,35 @@
+import base64
+import logging
+
+import pyAesCrypt
+
+from monkey_island.cc.server_utils.encryption import IEncryptor
+from monkey_island.cc.server_utils.encryption.encryptors.password_based_bytes_encryption import (
+    PasswordBasedBytesEncryptor,
+)
```

Comment:
can't do `from monkey_island.cc.server_utils.encryption import PasswordBasedBytesEncryptor` because of circular imports. 🤔